bindata/network/ovn-kubernetes/managed/alert-rules.yaml

```diff
@@ -316,6 +316,23 @@ spec:
       for: 15m
       labels:
         severity: warning
+    # OVN northbound and southbound database performance alerts
+    - alert: OVNKubernetesNorthboundDatabaseCPUUsageHigh
+      annotations:
+        summary: OVN northbound database {{ $labels.instance }} is greater than {{ $value }} percent CPU usage for a period of time.
+        description: High OVN northbound CPU usage indicates high load on the networking control plane.
+      expr: (sum(rate(container_cpu_usage_seconds_total{container="nbdb"}[5m])) BY (instance, name) * 100) > 80
```

Comment:
NBDB cpu should always be pretty low. Definitely less than 50%. @dceara do you agree? @mohit-sheth you may have some data to back this up.

Reply:
I agree.  NBDB cpu should be pretty low in most cases.

Reply:
Thats correct. On the scale cluster I got, I rarely saw nb db go over 30%. This NB DB alert is a bit pointless but  I added it anyway in case some issue could cause it to utilise a lot of CPU and this alert would capture it.